graphql_base/controllers/main.py

```diff
@@ -77,7 +61,7 @@ def _process_request(self, schema, data):
             return response
         except HttpQueryError as e:
             result = json_encode({"errors": [{"message": str(e)}]})
-            headers = dict(e.headers)
+            headers = dict()
```

Comment:
Why this change?

Reply:
The HttpQueryError does not contain any headers anymore

Reply:
I still see it [here](https://github.com/graphql-python/graphql-server/blob/master/graphql_server/error.py#L18), though. And this does not seem to have changed recently.